### DISCUSSION TITLE
vue-hooks-plus 打包后 lodash内依赖过大？

### DISCUSSION THREAD

**Discussion by kcmeven** [InhiblabCore/vue-hooks-plus#129] [General]
![image](https://user-images.githubusercontent.com/48147837/230806677-b23b1844-d2f7-4fae-bbdf-2fae397fd878.png)

**Comment by kcmeven** [comment]
![image](https://user-images.githubusercontent.com/48147837/230806831-d0b1ec8e-562f-44cc-b0b0-e1a8856c17c3.png)

**Reply by NelsonYong** [reply]
@k983551019 有一个地方全量引入了，稍后请关注最新版 `1.7.1`

**Reply by NelsonYong** [reply]
有问题可以 issue，discussions没提示，我也比较少看。🌸
